doc/cephadm/upgrade.rst

```diff
@@ -188,3 +188,43 @@ you need. For example, the following command upgrades to a development build:
   ceph orch upgrade start --image quay.io/ceph-ci/ceph:recent-git-branch-name
 
 For more information about available container images, see :ref:`containers`.
+
+
+Migrations
+==========
+
+.. _cephadm-migration-nfs:
+
+16.2.7 - NFS export migration
+-----------------------------
+
+This migration should be done fully automated, but in case something goes
+wrong while the upgrade is in progress, there is also a manual procedure
+
+in 16.2.6, the pool for storing NFS ganesha exports was arbitrary, after
+16.2.7, the pool name is always ``.nfs``. to create the Pool, run::
+
+  osd pool create pool .nfs
+  osd pool application enable .nfs nfs
+
+Previously, there was an extra parameter which was the RAODOS namespace. 
+After 16.2.7, the namespace is now always set to the cephadm service id. 
+
+Next, we have to copy over the ``grace`` object from the old location to
+the new location.
+
+To migrate export objects (all RADOS objects starting with ``export-``),
+run::
+
+  nfs export apply --cluster-id <cephadm-service-id> -i <local-path-to-export-object>
+
+with the old export object.
+
+.. note:: 
+
+  All lines containing ``secret_access_key`` or ``user-id`` should be 
```

Comment:
All lines where? In each of the export objects?

Reply:
are we sure about removing these? .. an empty user-id will default to `client.admin`

Reply:
https://github.com/ceph/ceph/blob/961bfc8e45f1279dc1348df5a9fefc451c53c8dd/src/pybind/mgr/cephadm/migrations.py#L250-L254

:shrug: